RiotSharpTest/TournamentRiotApiTest.cs

```diff
@@ -74,26 +71,19 @@ public void GetTournamentMatch_Test()
         [TestCategory("TournamentRiotApi"), TestCategory("Async")]
         public void CreateProviderAsync_CreateTournamentAsync_CreateTournamentCodeAsync_UpdateTournamentCodeAsync_Test()
```

Comment:
This test fails if you run it simultaneously with the others. However, it passes if you run it solo. I don't know why, though.